README.md

```diff
@@ -38,7 +38,9 @@ Create a client and use the public methods to access Proxmox resources.
 package main
 
 import (
+	"context"
```

Comment:
since you're already in this file could you add to the bullet list of features that context is fully supported?

Reply:
Sorry, kinda missed the boat for this one as my work day was already over at this point.

Reply:
no worries at all, figured it can always be done later and merged anyways to get the beta out while i was thinking about it. ill update the readme some day if you don't get around to it